Dockerfile

```diff
@@ -1,25 +1,31 @@
+FROM maven:3.6.1-jdk-8 AS tls-scanner-build
+
+RUN git clone --depth=1 https://github.com/RUB-NDS/TLS-Attacker.git && \
+	git clone --depth=1 --recurse-submodules https://github.com/RUB-NDS/TLS-Scanner.git && \
```

Comment:
And TLS-Scanner version 2.9: https://github.com/RUB-NDS/TLS-Scanner/tree/2.9

Because from version 3.0 they remove `-implementation` flag. I will update it later in code, but as for now, we need a little downgrade here.